components/servo/lib.rs

```diff
@@ -196,7 +196,7 @@ impl<Window> Browser<Window> where Window: WindowMethods + 'static {
         // as the navigation context.
         let (constellation_chan, sw_senders) = create_constellation(opts.user_agent.clone(),
                                                                     opts.config_dir.clone(),
-                                                                    opts.url.clone(),
+                                                                    Some(url),
```

Comment:
Can we not use an Option<> here? Is there any situation where we want to initialise a constellation without a url?